UnityProject/Assets/Scripts/Variable Viewer/BookViewer/ElementDisplay/SUB_ElementHandler.cs

```diff
@@ -23,4 +28,78 @@ public void ValueSetUp()
 		}
 	}
 
+	public void UpdateButtons()
+	{
+
+		var index = transform.GetSiblingIndex();
+		if (index == 0)
+		{
+			UpButton.gameObject.SetActive(false);
```

Comment:
`UpButton.gameObject.SetActive(index != 0);`

Reply:
no, you're looking at it wrong
0 = top of list

Reply:
oh sorry , Miss read  xD, I think it's clear like this though  since you can get the result of the calculation with the debugger

Reply:
oki